mailbox/store/src/main/java/org/apache/james/mailbox/store/StoreMessageResultIterator.java

```diff
@@ -1,348 +1,341 @@
-        public MailboxId getMailboxId() {
-            return mailboxId;
-        }
-
-        @Override
-        public Flags getFlags() {
-            return flags;
-        }
-
-        @Override
-        public Content getFullContent() throws MailboxException {
-            throw exception;
-        }
-
-        @Override
-        public Date getInternalDate() {
-            return internalDate;
-        }
-
-        @Override
-        public Content getBody() throws MailboxException {
-            throw exception;
-        }
-
-        @Override
-        public long getSize() {
-            return size;
-        }
-
-        @Override
-        public MessageUid getUid() {
-            return uid;
-        }
-
-        @Override
-        public MessageId getMessageId() {
-            return messageId;
-        }
-        
-        @Override
-        public int compareTo(MessageResult that) {
-            return uid.compareTo(that.getUid());
-        }
-
-        @Override
-        public int hashCode() {
-            return Objects.hashCode(exception, internalDate, size, uid, flags, modSeq, messageId);
-        }
-
-        @Override
-        public boolean equals(Object obj) {
-            if (this == obj) {
-                return true;
-            }
-            if (obj instanceof UnloadedMessageResult) {
-                UnloadedMessageResult that = (UnloadedMessageResult)obj;
-                return (size == that.size) && (uid.equals(that.uid)) && (modSeq == that.modSeq) && exception.equals(that.exception)
-                        && internalDate.equals(that.internalDate) && flags.equals(that.flags);
-            }
-            return false;
-        }
-
-        @Override
-        public Content getFullContent(MimePath path) throws MailboxException {
-            throw exception;
-        }
-
-        @Override
-        public Iterator<Header> iterateHeaders(MimePath path) throws MailboxException {
-            throw exception;
-        }
-
-        @Override
-        public Iterator<Header> iterateMimeHeaders(MimePath path) throws MailboxException {
-            throw exception;
-        }
-
-        @Override
-        public Content getBody(MimePath path) throws MailboxException {
-            throw exception;
-        }
-
-        @Override
-        public Content getMimeBody(MimePath path) throws MailboxException {
-            throw exception;
-        }
-
-        @Override
-        public MimeDescriptor getMimeDescriptor() throws MailboxException {
-            throw exception;
-        }
-
-        @Override
-        public long getModSeq() {
-            return modSeq;
-        }
-
-        @Override
-        public Headers getHeaders() throws MailboxException {
-            throw exception;
-        }
-
-        @Override
-        public List<MessageAttachment> getAttachments() throws MailboxException {
-            throw exception;
-        }
-
-    }
-
-}
+/****************************************************************
```

Comment:
you changed the indentation of that file (or at least it's what github shows) and it makes the review impossible ("hide whitespace changes" doesn't work)

Reply:
Crap must be `\\r\\n` issues. I am going to commit it separately.

Reply:
Yep that was this. Done in a separate commit.